frontend/src/lib/widgetTheme.ts

```diff
@@ -50,16 +56,52 @@ export const colors = {
   grayLight: SCSS_VARS["$gray-light"],
   grayLighter: SCSS_VARS["$gray-lighter"],
   grayLightest: SCSS_VARS["$gray-lightest"],
+  transparent: "transparent",
+
   primary: SCSS_VARS.$primary,
   primaryA50: SCSS_VARS["$primary-a50"],
-  transparent: "transparent",
+
+  secondary: SCSS_VARS.$secondary,
+  danger: SCSS_VARS.$danger,
+  disabledBg: SCSS_VARS.$disabled,
+  disabledColor: SCSS_VARS.$gray,
 }
 
 export const fontStyles = {
   fontFamily: fontFamilySans,
   fontSize: fontSizeBase,
+  fontSizeSm,
   fontWeight: "normal",
   lineHeight: lineHeightBase,
+  lineHeightTight,
+}
+
+export const variables = {
+  borderRadius,
+  spacer,
+}
+
+export enum Sizes {
+  SMALL = "sm",
+  MEDIUM = "md",
+  LARGE = "lg",
+  EXTRALARGE = "xl",
+}
```

Comment:
Any reason why this is here? Seems like these are associated with a specific component, and they should be in this case duplicated?

Reply:
It's fine for now, but I guess there might be exceptions to the rule (no xlarge progress bar for example). I won't make that big a deal.